chpl/chpl-service/src/main/java/gov/healthit/chpl/auth/user/JWTAuthenticatedUser.java

```diff
@@ -22,6 +23,8 @@
 public class JWTAuthenticatedUser implements UserDetails, Authentication {
     private static final long serialVersionUID = -7558546038256722930L;
 
+    @DeprecatedResponseField(message = "This field is deprecated and will be removed.", removalDate = "2024-11-01")
```

Comment:
I believe we usually give six months warning on deprecation

Reply:
This actually doesn't need `@DeprecatedResponseField`.  It is not an object that is publicly available and is only used internally within the API.